packages/lib/src/components/internal/ClickToPay/context/ClickToPayContext.ts

```diff
@@ -37,8 +39,10 @@ const ClickToPayContext = createContext<IClickToPayContext>({
     configuration: null,
     isStandaloneComponent: null,
     isCtpPrimaryPaymentMethod: null,
+    isStoringCookies: null,
```

Comment:
Should this be false? Type says `boolean`. Otherwise `undefined` might be better.

Reply:
Sure. Somehow Typescript does not complain about it, even when informing it that this value is of type `boolean`.